src/proof_system/widget/arithmetic/proverkey.rs

```diff
@@ -61,7 +61,7 @@ impl ProverKey {
         let q_c_poly = &self.q_c.0;
         let q_4_poly = &self.q_4.0;
 
-        // (a_eval * b_eval * q_m_poly + a_eval * q_l + b_eval * q_r + c_eval * q_o + d_eval * q_4 + q_c) * q_arith_eval * alpha
+        // (a_eval * b_eval * q_m_poly + a_eval * q_l + b_eval * q_r + c_eval * q_o + d_eval * q_4 + q_c) * q_arith_eval
```

Comment:
This is correct to change. The reason alpha stayed in the docs but was removed from the code is we were going to increase powers of alpha to create custom gates. However, when there were many different types of gates in the end, we ended up creating random `separation challenges`.